packages/@react-spectrum/tag/src/TagGroup.tsx

```diff
@@ -87,7 +72,7 @@ function TagGroup<T extends object>(props: SpectrumTagGroupProps<T>, ref: DOMRef
 
   // Don't want the grid to be focusable or accessible via keyboard
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  let {tabIndex, ...otherGridProps} = gridProps;
+  let {tabIndex, role, ...otherGridProps} = gridProps;
```

Comment:
I noticed that Home/End/PageUp/PageDown don't move focus accordingly. Home/End should move to the first/last tag, not entirely sure how PageUp/PageDown should function (v2 has PageUp/Down move to the previous/next tag which is what Up/Down does right now). I encountered the same issue in ListView, you can override `getFirstKey` and `getLastKey` in the TagKeyboardDelegate to return the first/last tag in the collection to fix this.

Reply:
Similarly, should focus wrap when using the arrow keys (v2 TagList parity)? aka If I keep pressing the Left arrow, should focus go from the first tag to the last tag? Lemme know if we had already discussed this and decided that it shouldn't do this

Reply:
On a separate note, did you get a chance to try "GridProps: potentially don't need to destructure tabIndex out"?

Reply:
- i think maintaining v2 parity would be the correct path. i don't think it was talked through in previous conversations
- i wasn't able to find a way to leave `role` in gridProps and maintain correct keyboard nav/focus